src/scripts/main.js

```diff
@@ -1,3 +1,33 @@
 'use strict';
 
 // write your code here
+const logo = document.querySelector('.logo');
+
+const promise1 = new Promise((resolve) => {
+  logo.addEventListener('click', () => {
+    resolve('Promise was resolved!');
+  });
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(reject, 3000);
+});
+
+function onSuccess() {
+  const successMessage = document.createElement('div');
+
+  successMessage.className = 'message';
+  successMessage.textContent = 'Promise was resolved!';
+  document.body.append(successMessage);
+}
+
+function onError() {
+  const errorMessage = document.createElement('div');
+
+  errorMessage.classList.add('message', 'error-message');
+  errorMessage.textContent = 'Promise was rejected!';
+  document.body.append(errorMessage);
+}
+
+promise1.then(onSuccess).catch(onError);
+promise2.then(onSuccess).catch(onError);
```

Comment:
The current implementation handles `promise1` and `promise2` independently. This means that if `promise2` rejects after 3 seconds, it will trigger the `onError` function even if `promise1` has already resolved successfully. If the intention is to wait for either promise to settle and then take action, consider using `Promise.race` or `Promise.allSettled` to handle them together.